load-test-framework/src/main/java/com/google/pubsub/flic/Driver.java

```diff
@@ -189,6 +202,10 @@ private void run() {
       Client.maxOutstandingRequests = maxOutstandingRequests;
       Client.burnInTimeMillis = (Client.startTime.getSeconds() + burnInDurationSeconds) * 1000;
       Client.numberOfMessages = numberOfMessages;
+      Client.cpsPublisherCount = cpsPublisherCount;
```

Comment:
I'm going to change this because I think the abstraction/design point is valid, but I have a question to clarify the O(2n) space usage. I made all the publisher/subscriber count variables static in the Client class, so my thinking there was it's stored once for the class per jvm. Is that right, or am I not understanding how java does that correctly?